root/layout/components/sidebar/ArtistSidebar.js

```diff
@@ -72,9 +78,9 @@ const ArtistSidebar = ({$c, artist}: Props) => {
 
         <SidebarType entity={artist} typeType="artist_type" />
 
-        {artist.gender ? (
+        {gender ? (
           <SidebarProperty className="gender" label={l('Gender:')}>
-            {l(artist.gender.name)}
```

Comment:
Flow doesn't like this anymore because a function is called inside an attribute (`l('Gender:')`). At run-time that function will be called before anything in the children (here `l(artist.gender.name)`), and Flow can't prove that it doesn't, say, set `artist.gender` to null before this line runs, so it invalidates the refinement. Which seems somewhat silly in this case, but it's trying to be as sound as possible.

On the other hand, if these were reversed (with `label={l(artist.gender.name)}` as the attribute and `{l('Gender:')}` as the child), Flow won't invalidate the refinement for the attribute anymore just because you call a function in the children. (It did this in 0.113 and earlier, which was a bug.) So it's better in some ways.

Declaring the variables above as `const` works around all of this, because Flow knows a const variable can't be reassigned.